certGenerator.go

```diff
@@ -0,0 +1,46 @@
+// based on https://stackoverflow.com/q/26441547/1426535
+
```

Comment:
This might be overkill, but I'd like to see this gated behind a [build contraint tag](https://golang.org/cmd/go/#hdr-Build_constraints), similar to the integration tests.

I don't think we want to distribute this CLI program (e.g. in a ZLint release).